src/frontend/sharedComponents/LocationField.js

```diff
@@ -55,6 +58,7 @@ const LocationField = ({ children }: Props) => {
   return children({
     longitude: value.lon,
     latitude: value.lat,
+    system: coordinateSystem,
```

Comment:
This doesn't really belong here, because this component has nothing to do with the display of coordinates. To separate concerns, I think it's better to keep this in the `<LocationView>` component